### ISSUE TITLE
Missing knobs

### ISSUE THREAD

**Issue by Daniele71** [psemiletov/drumrox#12]
Hello, version from openSUSE repo is outdated so I made my own package (just changed the version number inside the spec file..).
Now the knobs are missing.  Do I need the cmke option about knobs ?
![drumrox](https://github.com/psemiletov/drumrox/assets/40473719/de7f5dd4-83fa-4f32-83f8-382f79fbfe1d)

**Comment by psemiletov** [comment]
It's ok because there are no internal pan and level handling in multi-channel mode. Use track pan/level controls instead.

**Comment by Daniele71** [comment]
> It's ok because there are no internal pan and level handling in multi-channel mode. Use track pan/level controls instead.

Uhm.. ok make some sense but it's a bit confusing. Looking at the GUI, it really seems like that  something is missing. LSP-multisampler the GUI is exactly the same for the stereo and the multiout version.  If there's no technical problem I would think a bit about it.  Maybe add something to the README...
Anyway, thanks for the plugin, now the version in the openSUSE proaudio repo is updated..

**Comment by psemiletov** [comment]
Yes, you right - I'll update the documentation. By the way, my recent efforts is another drum machine - Drumlabooh (https://psemiletov.github.io/drumlabooh/). It works like Drumrox, but written from scratch and compatible with Reaper and other DAW's.

**Comment by Daniele71** [comment]
> Yes, you right - I'll update the documentation. By the way, my recent efforts is another drum machine - Drumlabooh (https://psemiletov.github.io/drumlabooh/). It works like Drumrox, but written from scratch and compatible with Reaper and other DAW's.

does it have multi out? Drum sampler without multi output are non useable for me..
